pkg/kube/activity.go

```diff
@@ -29,6 +30,7 @@ type PipelineActivityKey struct {
 	LastCommitMessage string
 	LastCommitURL     string
 	GitInfo           *gits.GitRepository
+	PullRefs          map[string]string
```

Comment:
Would it make sense to have some `builtType` as well?

Reply:
I think you can get the same result by checking the length of the `PullRefs`. I'm already creating a function to return the number of refs of a `PipelineActivityKey`. 

Adding a buildType property would just imply moving that len check to somewhere else just to initialize the variable.

Considering the length of PullRefs is being used elsewhere to do other things, I don't think it's necessary.

Reply:
ok